azurerm/resource_arm_service_fabric_cluster_test.go

```diff
@@ -721,6 +754,47 @@ resource "azurerm_service_fabric_cluster" "test" {
 `, rInt, location, rInt)
 }
 
+func testAccAzureRMServiceFabricCluster_azureActiveDirectory(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_service_fabric_cluster" "test" {
+  name                = "acctest-%d"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  location            = "${azurerm_resource_group.test.location}"
+  reliability_level   = "Bronze"
+  upgrade_mode        = "Automatic"
+  vm_image            = "Windows"
+  management_endpoint = "https://example:80"
+	
+	azure_active_directory {
+		tenant_id              = "00000000-0000-0000-0000-000000000000"
+		cluster_application_id = "00000000-0000-0000-0000-000000000000"
+		client_application_id  = "00000000-0000-0000-0000-000000000000"
+	}
```

Comment:
_minor_ could we fix the indentation for this block to match the rest of this (using spaces instead of tabs?)

Reply:
Thanks, resolved indentation error.